### PR TITLE
Size token updates

### PR DESCRIPTION
**Pull request by libertymayc**
Deprecate `--salt-size-graphic-` tokens
Simplify calculation for `--salt-size-divider-height`

### CODE REVIEW

File: .changeset/odd-wasps-eat.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+"@salt-ds/lab": minor
+"@salt-ds/theme": minor
+---
+
+Remove `--salt-size-graphic-small`, `--salt-size-graphic-medium`, `--salt-size-graphic-large`
```
Comment (joshwooding) [inline .changeset/odd-wasps-eat.md:6]: Changeset needs to be updated

File: packages/theme/css/foundations/size.css [hunk 2]
```diff
@@ -10,18 +10,16 @@
...
-  --salt-size-graphic-medium: 24px;
-  --salt-size-graphic-large: 48px;
-
   /* Deprecated */
   --salt-size-selection: var(--salt-size-selectable);
-  --salt-size-brandBar: 4px; /* Use --salt-size-accent */
+  --salt-size-brandBar: 4px; /* Use --salt-size-accent */  
+  --salt-size-graphic-small: 12px; 
+  --salt-size-graphic-medium: 24px;
+  --salt-size-graphic-large: 48px;
```
[the remark this answers is not in the document]
Comment (joshwooding) [reply packages/theme/css/foundations/size.css:22]: Tricky; I would say not to remove them just to be consistent.

**Review by joshwooding — CHANGES_REQUESTED** [review]

File: .changeset/odd-wasps-eat.md (hunk 1 above)
Comment (libertymayc) [inline .changeset/odd-wasps-eat.md:6]:
```suggestion
Deprecated `--salt-size-graphic-small`, `--salt-size-graphic-medium`, `--salt-size-graphic-large`
```

File: .changeset/odd-wasps-eat.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+---
+"@salt-ds/lab": minor
+"@salt-ds/theme": minor
+---
+
+Deprecated `--salt-size-graphic-small`, `--salt-size-graphic-medium`, `--salt-size-graphic-large`
```
Comment (libertymayc) [inline .changeset/odd-wasps-eat.md:6]:
```suggestion
Deprecated `--salt-size-graphic-small`, `--salt-size-graphic-medium`, `--salt-size-graphic-large`
Simplified `--salt-size-compact` calculation from `calc(var(--salt-size-base) - (1.5 * var(--salt-size-unit)) - (0.5 * var(--salt-size-basis-unit)))` to `calc(var(--salt-size-compact) + 1.5 * var(--salt-size-basis-unit))`
```

**Pull request merged**